### DISCUSSION TITLE
What happened to the FPS option in custom image quality?

### DISCUSSION THREAD

**Discussion by discoveryX64** [rustdesk/rustdesk#5923] [General]
Title

**Comment by discoveryX64** [comment]
<img width="494" alt="Screenshot 2023-10-05 at 9 18 23 AM" src="https://github.com/rustdesk/rustdesk/assets/119447512/d60f6481-90e3-4582-bdc0-0b3bc25147b8">
<img width="506" alt="Screenshot 2023-10-05 at 9 18 16 AM" src="https://github.com/rustdesk/rustdesk/assets/119447512/f73c2c3c-74fb-42e9-a5df-5bd6bf27e675">

**Reply by 21pages** [reply]
https://github.com/rustdesk/rustdesk/blob/53cd259ffa21ac9c3b2d6a25ed088c6e0c4e000b/flutter/lib/common/widgets/dialog.dart#L1297

**Reply by discoveryX64** [reply]
So if I delete/change that setting it will work again?

**Reply by 21pages** [reply]
> So if I delete/change that setting it will work again?

It shows when self host or direct connection.

**Reply by discoveryX64** [reply]
So if it's direct connection it doesn't show up and when it is it does?

**Reply by discoveryX64** [reply]
or vise-versa

**Reply by 21pages** [reply]
If it's direct connection and the rustdesk's version on the controlled side is bigger than 1.2.0, it shows fps option.

**Reply by discoveryX64** [reply]
Both versions are 1.2.3 nightly though. And I have direct connection on both.

**Reply by 21pages** [reply]
![1697587738544](https://github.com/rustdesk/rustdesk/assets/14891774/f1bbc015-4a0f-4636-bfc2-42129eb0b387)
